components/quick_input.jsx

```diff
@@ -24,11 +28,22 @@ export default class QuickInput extends React.PureComponent {
          * The string value displayed in this input
          */
         value: PropTypes.string.isRequired,
+
+        /**
+         * Whether it shows an X on the input field that clears the input when clicked. Default: false
+         */
+        clearable: PropTypes.bool,
+
+        /**
+         * An optional function to handle clearing the input field when the X is clicked.
+         */
+        handleClear: PropTypes.func,
```

Comment:
Thanks for the comment @ali-farooq0 . You are right, but it still feels a bit weird. "handleClear" means handle the clear action. Therefore, instead of onHandleClear, would be onClear. Would that make more sense? Or do you still prefer onHandleClear? Thanks!

Reply:
`onClear` works as well. And I agree, i was thinking the same thing i.e. `handleClear` implicitly denotes the caller to handle the clear functionality, which is not the case. +1 for `onClear`.